src/main/scala/edu/ie3/powerFactory2psdm/config/ConversionConfig.scala

```diff
@@ -6,52 +6,146 @@
+    *                     which uses the apparent power mentioned in the models basic data (Basisdaten) or
+    *                     [[loadFlowSource]] which uses the apparent power of the load flow (Lastfluss) specification
+    * @param cosPhiSource which cosinus phi source to choose from the PowerFactory model. Can be either [[basicDataSource]]
+    *                     which uses the rated cosinus phi mentioned in the models basic data (Basisdaten) or
+    *                     [[loadFlowSource]] which uses the cosinus phi of the load flow (Lastfluss) specification
+    *
+    */
+  case class StatGenModelConfigs(
+      pvConfig: PvConfig,
+      sRatedSource: String,
+      cosPhiSource: String
   )
 
+  /**
+    * General configuration for pv model conversion of static generators of category "Fotovoltaik".
+    * As the model does not include most of the model parameters we can choose by setting the [[conversionMode]] to
+    * either convert the models to [[edu.ie3.datamodel.models.input.system.FixedFeedInInput]] or generate a
+    * [[edu.ie3.datamodel.models.input.system.PvInput]] by sampling the missing parameters.
+    *
+    * @param conversionMode convert to fixed feed-in or generate model
+    * @param individualConfigs for certain generators
+    */
   case class PvConfig(
       conversionMode: PvConversionMode,
       individualConfigs: Option[List[IndividualPvConfig]]
   )
 
+  /**
+    * Individual configuration for conversion of the set of generators with the given [[ids]]
+    *
+    * @param ids models to apply the [[conversionMode]] to
+    * @param conversionMode to apply for the models
+    */
   case class IndividualPvConfig(
       ids: Set[String],
       conversionMode: PvConversionMode
   )
 
+  /**
+    * Trait to denote modes for converting pv static generators
+    */
   sealed trait PvConversionMode
 
+  /**
+    * Convert to a [[edu.ie3.datamodel.models.input.system.FixedFeedInInput]]
+    */
   case object PvFixedFeedIn extends PvConversionMode
 
+  /**
+    * Convert to a [[edu.ie3.datamodel.models.input.system.PvInput]] considering the given methods for generating the
+    * missing parameters.
+    *
+    * @param albedo Albedo value (typically a value between 0 and 1)
+    * @param azimuth Inclination in a compass direction (typically °: South 0◦; West 90◦; East -90◦)
+    * @param height Tilted inclination from horizontal (typically in °)
+    * @param etaConv Efficiency of converter (typically in %)
+    * @param qCharacteristic Description of a reactive power characteristic
+    * @param kG Generator correction factor merging different technical influences
+    * @param kT Generator correction factor merging different technical influences
+    */
   case class PvModelGeneration(
       albedo: GenerationMethod,
       azimuth: GenerationMethod,
+      height: GenerationMethod,
```

Comment:
I do but I thought that might get confusing when the parameters here don't match the PvInput parameters. But since Murad is tackling this issue I might also introduce the change here.